internal/k8s/controller.go

```diff
@@ -2697,6 +2928,30 @@ func getWAFPoliciesForAppProtectLogConf(pols []*conf_v1.Policy, key string) []*c
 	return policies
 }
 
+func getBadosPoliciesForAppProtectDosPolicy(pols []*conf_v1.Policy, key string) []*conf_v1.Policy {
+	var policies []*conf_v1.Policy
+
+	for _, pol := range pols {
+		if pol.Spec.Bados != nil && isMatchingResourceRef(pol.Namespace, pol.Spec.Bados.ApDosPolicy, key) {
+			policies = append(policies, pol)
+		}
+	}
+
+	return policies
+}
+
+func getBadosPoliciesForAppProtectLogConf(pols []*conf_v1.Policy, key string) []*conf_v1.Policy {
```

Comment:
Could we have a unit test similar to TestGetWAFPoliciesForAppProtectLogConf?

Reply:
done